components/ws-manager-api/core.proto

```diff
@@ -232,6 +232,9 @@ message DeleteVolumeSnapshotRequest{
 
     // soft_delete controls whether manager should attempt to restore volume snapshot from handle if it doesn't exist in the cluster yet
     bool soft_delete = 3;
+
+    // ws_type is the type of workspace (prebuild or regular) to which this volume snapshot belongs
+    string ws_type = 4;
```

Comment:
Should this be an enum? It's a private proto so we don't have to worry about supporting future types (we can always extend the enum) and it provides much higher safety when crossing the system boundary.

Reply:
Fixed! Thank you for suggestion!